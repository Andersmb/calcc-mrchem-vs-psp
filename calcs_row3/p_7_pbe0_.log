Starting job 5688406 on c1-[24,36,38],c2-49,c3-18,c5-49 at Fri May 20 17:12:38 CEST 2022

Copying p_7_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying p_7_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688406       p_7_pbe0_        180                                             00:41:19      0:0 
5688406.bat+      batch         36        1   00:00:03          0   00:00:03   00:41:19      0:0 
5688406.ext+     extern        180        6   00:00:00          0   00:00:00   00:41:19      0:0 
5688406.0      mrchem.x        180       30   00:39:12         29   01:38:59   00:39:18      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688406                                                                          
5688406.bat+     19332K          0     19332K        0              0          0 
5688406.ext+          0          0          0        0              0          0 
5688406.0     13305908K         29 4817541939        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688406                                                                                               
5688406.bat+       19.44M               0         19.44M        0.07M                0          0.07M 
5688406.ext+        0.00M               0          0.00M            0                0              0 
5688406.0           6.77M               0          6.74M      129.73M                3         64.10M 

Job 5688406 completed at Fri May 20 17:53:58 CEST 2022
