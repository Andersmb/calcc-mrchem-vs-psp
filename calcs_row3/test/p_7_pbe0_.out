                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        1.073455        0.000000
    1       F            :        0.000000       -0.875042        2.260490
    2       F            :        0.000000       -0.875042       -2.260490
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              16
 Total electrons         :                                              33
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999826812e-01
    1    1    a          :                              9.999999656591e-01
    2    1    a          :                              1.000000008204e+00
    3    1    a          :                              9.999999918507e-01
    4    1    a          :                              1.000000003026e+00
    5    1    a          :                              9.999999984869e-01
    6    1    a          :                              9.999999999893e-01
    7    1    a          :                              9.999999974597e-01
    8    1    a          :                              9.999999676671e-01
    9    1    a          :                              9.999993791489e-01
   10    1    a          :                              1.000000889105e+00
   11    1    a          :                              9.999993566692e-01
   12    1    a          :                              1.000000046981e+00
   13    1    a          :                              1.000001939091e+00
   14    1    a          :                              1.000000017110e+00
   15    1    a          :                              1.000000122953e+00
   16    1    a          :                              1.000000010938e+00
   17    1    b          :                              1.000000079531e+00
   18    1    b          :                              9.999998920476e-01
   19    1    b          :                              1.000000168078e+00
   20    1    b          :                              1.000000004722e+00
   21    1    b          :                              1.000000003034e+00
   22    1    b          :                              9.999999992715e-01
   23    1    b          :                              9.999999999786e-01
   24    1    b          :                              9.999999975969e-01
   25    1    b          :                              9.999999905200e-01
   26    1    b          :                              9.999993795598e-01
   27    1    b          :                              1.000000892539e+00
   28    1    b          :                              9.999993634357e-01
   29    1    b          :                              1.000000050311e+00
   30    1    b          :                              1.000000830964e+00
   31    1    b          :                              9.999999409429e-01
   32    1    b          :                              1.000000030076e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                535.944312759879
 E-N energy              :            (au)              -1498.970732647491
 Coulomb energy          :            (au)                360.754793298970
 Exchange energy         :            (au)                -10.716651288006
 X-C energy              :            (au)                -33.230555361511
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                108.387872884532
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -646.218833238159
 Nuclear energy          :            (au)                108.387872884532
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.378309603536e+02
                         :      (kcal/mol)             -3.374940203433e+05
                         :        (kJ/mol)             -1.412074982033e+06
                         :            (eV)             -1.463512535399e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.893515236557
    1    1    a          :            (au)                -24.521241526985
    2    1    a          :            (au)                -24.518328692547
    3    1    a          :            (au)                 -6.748314227695
    4    1    a          :            (au)                 -4.888537785417
    5    1    a          :            (au)                 -4.886698318536
    6    1    a          :            (au)                 -4.883509528704
    7    1    a          :            (au)                 -1.233255955843
    8    1    a          :            (au)                 -1.185137481748
    9    1    a          :            (au)                 -0.635387685631
   10    1    a          :            (au)                 -0.476402435013
   11    1    a          :            (au)                 -0.470485063550
   12    1    a          :            (au)                 -0.452560122246
   13    1    a          :            (au)                 -0.388573798582
   14    1    a          :            (au)                 -0.368614455112
   15    1    a          :            (au)                 -0.345846261661
   16    1    a          :            (au)                 -0.217036906682
   17    1    b          :            (au)                -76.889674980969
   18    1    b          :            (au)                -24.517904228744
   19    1    b          :            (au)                -24.514899629466
   20    1    b          :            (au)                 -6.743935634260
   21    1    b          :            (au)                 -4.886285620138
   22    1    b          :            (au)                 -4.881289417292
   23    1    b          :            (au)                 -4.871777366898
   24    1    b          :            (au)                 -1.224102313603
   25    1    b          :            (au)                 -1.176928906178
   26    1    b          :            (au)                 -0.611328909051
   27    1    b          :            (au)                 -0.471003172115
   28    1    b          :            (au)                 -0.444082451995
   29    1    b          :            (au)                 -0.428758680199
   30    1    b          :            (au)                 -0.365172858209
   31    1    b          :            (au)                 -0.360373770417
   32    1    b          :            (au)                 -0.332618970389
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -305.833582392429
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.744563e+00        -537.830960353627         -5.378310e+02
    1          3.469349e-01        -540.730044969639         -2.899085e+00
    2          1.419863e-01        -540.841601985068         -1.115570e-01
    3          8.570897e-02        -540.861885944751         -2.028396e-02
    4          4.657062e-02        -540.868340863489         -6.454919e-03
    5          5.647056e-02        -540.873350647209         -5.009784e-03
    6          1.786625e-02        -540.873976710125         -6.260629e-04
    7          1.788444e-02        -540.874437837051         -4.611269e-04
    8          6.391562e-03        -540.874501746244         -6.390919e-05
    9          5.919868e-03        -540.874538174911         -3.642867e-05
   10          2.355282e-03        -540.874548040001         -9.865090e-06
   11          2.020020e-03        -540.874554069257         -6.029256e-06
   12          8.199450e-04        -540.874555241519         -1.172262e-06
   13          4.452968e-04        -540.874555651096         -4.095766e-07
   14          3.686830e-04        -540.874555858346         -2.072500e-07
   15          2.577645e-04        -540.874555952252         -9.390624e-08
   16          1.647547e-04        -540.874555990786         -3.853427e-08
   17          1.018240e-04        -540.874556003230         -1.244348e-08
---------------------------------------------------------------------------
                      SCF converged in 17 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        1.073455        0.000000
    1       F            :        0.000000       -0.875042        2.260490
    2       F            :        0.000000       -0.875042       -2.260490
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                539.851228410766
 E-N energy              :            (au)              -1505.907124581119
 Coulomb energy          :            (au)                360.691550773273
 Exchange energy         :            (au)                -10.694773703286
 X-C energy              :            (au)                -33.203309787395
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                108.387872884532
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -649.262428887761
 Nuclear energy          :            (au)                108.387872884532
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.408745560032e+02
                         :      (kcal/mol)             -3.394039054332e+05
                         :        (kJ/mol)             -1.420065941254e+06
                         :            (eV)             -1.471794580715e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.318784326179
    1    1    a          :            (au)                -24.808358084605
    2    1    a          :            (au)                -24.808354685039
    3    1    a          :            (au)                 -6.707353478953
    4    1    a          :            (au)                 -4.845239330880
    5    1    a          :            (au)                 -4.844168414412
    6    1    a          :            (au)                 -4.840046551415
    7    1    a          :            (au)                 -1.293579408193
    8    1    a          :            (au)                 -1.260770975936
    9    1    a          :            (au)                 -0.682126335905
   10    1    a          :            (au)                 -0.569097929045
   11    1    a          :            (au)                 -0.534280695606
   12    1    a          :            (au)                 -0.531553741622
   13    1    a          :            (au)                 -0.492697390602
   14    1    a          :            (au)                 -0.467113446045
   15    1    a          :            (au)                 -0.401426676204
   16    1    a          :            (au)                 -0.236789809082
   17    1    b          :            (au)                -77.314342173756
   18    1    b          :            (au)                -24.805475724603
   19    1    b          :            (au)                -24.805471468940
   20    1    b          :            (au)                 -6.702701182636
   21    1    b          :            (au)                 -4.841609897108
   22    1    b          :            (au)                 -4.838125518101
   23    1    b          :            (au)                 -4.828638585009
   24    1    b          :            (au)                 -1.287229168572
   25    1    b          :            (au)                 -1.254920714574
   26    1    b          :            (au)                 -0.663722092484
   27    1    b          :            (au)                 -0.565338952489
   28    1    b          :            (au)                 -0.527636456698
   29    1    b          :            (au)                 -0.510860032871
   30    1    b          :            (au)                 -0.474645138186
   31    1    b          :            (au)                 -0.465259656496
   32    1    b          :            (au)                 -0.375234769946
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -308.902952812192
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000017        0.003922       -0.000004
 Magnitude               :            (au)                        0.003922
                         :         (Debye)                        0.009968
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.351076        0.000000
 Magnitude               :            (au)                        0.351076
                         :         (Debye)                        0.892345
---------------------------------------------------------------------------
 Total vector            :       -0.000017        0.354997       -0.000004
 Magnitude               :            (au)                        0.354997
                         :         (Debye)                        0.902313
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 39m 13s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
